src/cpu/matmul/cpu_matmul_list.cpp

```diff
@@ -41,6 +41,12 @@ using namespace dnnl::impl::cpu::x64;
 #endif
 using namespace dnnl::impl::cpu::aarch64::matmul;
 using namespace dnnl::impl::cpu::aarch64;
+#elif DNNL_RV64
+#if DNNL_RISCV_USE_RVV_INTRINSICS
```

Comment:
The value is not provided for this macro.
```suggestion
#ifdef DNNL_RISCV_USE_RVV_INTRINSICS
```